src/components/Upload/Cropper/EasyCrop.tsx

```diff
@@ -54,6 +54,7 @@ const EasyCrop = forwardRef<EasyCropHandle, EasyCropProps>((props, ref) => {
   const [cropSize, setCropSize] = useState<Size>({ width: 0, height: 0 });
   const [zoomVal, setZoomVal] = useState<number>(INIT_ZOOM);
   const [rotateVal, setRotateVal] = useState<number>(INIT_ROTATE);
+  const [buttonType, setButtonType] = useState<string>('');
```

Comment:
can we set this type as possible/expected values? `'in' | 'out' | 'left' | 'right'`

Reply:
Yes. Updating this right away.